<HTML>
<HEAD>
<meta charset="UTF-8">
<title>net.corda.client.rpc.reconnect - corda</title>
<link rel="stylesheet" href="../../style.css">
</HEAD>
<BODY>
<a href="../index.html">corda</a>&nbsp;/&nbsp;<a href="./index.html">net.corda.client.rpc.reconnect</a><br/>
<br/>
<h2>Package net.corda.client.rpc.reconnect</h2>
<h3>Exceptions</h3>
<table>
<tbody>
<tr>
<td>
<h4><a href="-could-not-start-flow-exception/index.html">CouldNotStartFlowException</a></h4>
</td>
<td>
<p>Thrown when a flow start command died before receiving a <a href="#">net.corda.core.messaging.FlowHandle</a>.
On catching this exception, the typical behaviour is to run a "logical retry", meaning only retry the flow if the expected outcome did not occur.</p>
<code><span class="keyword">class </span><span class="identifier">CouldNotStartFlowException</span>&nbsp;<span class="symbol">:</span>&nbsp;<a href="../net.corda.client.rpc/-r-p-c-exception/index.html"><span class="identifier">RPCException</span></a></code></td>
</tr>
</tbody>
</table>
</BODY>
</HTML>
